src/BloomBrowserUI/bookEdit/readerSetup/ReaderSetup.jade

```diff
@@ -1,189 +0,0 @@
-doctype html
```

Comment:
??? Not sure why this file appears to be going away... I don't think I did that!?